src/components/button/Button.tsx

```diff
@@ -92,11 +99,12 @@ const getButtonVariables = ({
 
 const Button = styled(ButtonBase)<PButton>`
     align-items: stretch;
+    cursor: pointer;
 
     // define local variables
     ${getButtonVariables};
 
-    transition: all 500ms 0s ease-in-out;
+    transition: box-shadow 150ms 0s ease-in-out;
```

Comment:
I know this still needs to be worked on, but having background-color and color here is probably also a good idea. :)